review/communication-react.api.md

```diff
@@ -0,0 +1,1828 @@
+    chatMessages: {
+        [key: string]: ChatMessageWithStatus;
+    };
+    participants: {
+        [key: string]: ChatParticipant;
+    };
+    threadId: string;
+    properties?: ChatThreadProperties;
+    readReceipts: ChatMessageReadReceipt[];
+    typingIndicators: TypingIndicatorReceivedEvent[];
+    latestReadTime: Date;
+};
+
+// @public (undocumented)
+export type ChatThreadProperties = {
+    topic?: string;
+};
+
+// @public (undocumented)
+export const chatThreadSelector: OutputParametricSelector<ChatClientState, ChatBaseSelectorProps, {
+userId: string;
+showMessageStatus: boolean;
+messages: (Message<"chat"> | Message<"system"> | Message<"custom">)[];
+}, (res1: string, res2: {
+[key: string]: ChatMessageWithStatus;
+}, res3: Date, res4: boolean) => {
+userId: string;
+showMessageStatus: boolean;
+messages: (Message<"chat"> | Message<"system"> | Message<"custom">)[];
+}>;
+
+// @public (undocumented)
+export type ChatUIState = {
+    error?: Error;
+};
+
+// @public (undocumented)
+export type ClientState = CallClientState & ChatClientState;
+
+// @public (undocumented)
+export type Common<A, B> = Pick<A, CommonProperties<A, B>>;
+
+// @public (undocumented)
+export type CommonProperties<A, B> = {
+    [P in keyof A & keyof B]: A[P] extends B[P] ? P : never;
+}[keyof A & keyof B];
+
+// @public
+export type CommunicationParticipant = {
+    userId: string;
+    displayName?: string;
+};
+
+// @public
+export interface ComponentStrings {
+    cameraButton: CameraButtonStrings;
+    endCallButton: EndCallButtonStrings;
+    errorBar: ErrorBarStrings;
+    messageStatusIndicator: MessageStatusIndicatorStrings;
+    messageThread: MessageThreadStrings;
+    microphoneButton: MicrophoneButtonStrings;
+    optionsButton: OptionsButtonStrings;
+    participantItem: ParticipantItemStrings;
+    participantsButton: ParticipantsButtonStrings;
+    screenShareButton: ScreenShareButtonStrings;
+    sendBox: SendBoxStrings;
+    typingIndicator: TypingIndicatorStrings;
+}
+
+// @public
+export const ControlBar: (props: ControlBarProps) => JSX.Element;
+
+// @public
+export const ControlBarButton: (props: ControlBarButtonProps) => JSX.Element;
+
+// @public
+export interface ControlBarButtonProps extends IButtonProps {
+    labelKey?: string;
+    onRenderOffIcon?: IRenderFunction<IButtonProps>;
+    onRenderOnIcon?: IRenderFunction<IButtonProps>;
+    showLabel?: boolean;
+    strings?: ControlBarButtonStrings;
+}
+
+// @public
+export interface ControlBarButtonStrings {
+    label?: string;
+    offLabel?: string;
+    onLabel?: string;
+}
+
+// @public (undocumented)
+export type ControlBarLayoutType = 'horizontal' | 'vertical' | 'dockedTop' | 'dockedBottom' | 'dockedLeft' | 'dockedRight' | 'floatingTop' | 'floatingBottom' | 'floatingLeft' | 'floatingRight';
+
+// @public
+export interface ControlBarProps {
+    children?: React_2.ReactNode;
+    layout?: ControlBarLayoutType;
+    styles?: BaseCustomStylesProps;
```

Comment:
`BaseCustomStylesProps` is very generic, should this be whatever the control bar parent actually is? e.g. `IStackStyles`

Reply:
Looks like this one only contains a root property, which is more accurate for this case:
![image](https://user-images.githubusercontent.com/11863655/133197824-237e4ba0-3314-450a-b8bf-4e5293896f0f.png)